src/main/webapp/app/exam/participate/exam-participation.component.ts

```diff
@@ -324,7 +324,12 @@ export class ExamParticipationComponent implements OnInit, OnDestroy, ComponentC
         // If we don't receive a response within that time throw an error the subscription can then handle
         this.examParticipationService
             .submitStudentExam(this.courseId, this.examId, this.studentExam)
-            .pipe(timeoutWith(20000, throwError(new Error('Submission request timed out. Please check your connection and try again.'))))
+            .pipe(
+                timeoutWith(
```

Comment:
You might also want to replace the timeout deprecation here.

Reply:
I think I did that in #4609, but now definitely in commit 85a4e0e0897b7a86171f5cac6690e69db2c3132e.